### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by NikolayGit1**
- [DEMO LINK](https://nikolaygit1.github.io/layout_catalog/)
- [TEST REPORT LINK](https://nikolaygit1.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,454 @@
...
+          <li class="nav__item">
+            <a
+              href="#APPLE"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
```
Comment (alexandra-protyanova) [inline src/index.html:36]: when u have 2 atributes leave them on 1 line

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,454 @@
...
+          alt="screen"
+        >
+        <h2 class="list">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+        <p class="product">
+          Код товара: 195434
+        </p>
+        <div
+          class="stars stars--4"
```
Comment (alexandra-protyanova) [inline src/index.html:157]: here too

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,227 @@
-body {
+.body {
   margin: 0;
+  background-color: white;
 }
+
+//  #region moyo
```
Comment (alexandra-protyanova) [inline src/styles/main.scss:6]: split your scss to separate files, by BEM blocks

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
your code is not splitted into separate files (1 file per BEM block)

**Pull request closed without merging**